src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Square extends Figure {
+    private int side;
+
+    public Square() {
+        super();
+        this.side = new Random().nextInt(100);
```

Comment:
Random value generation should be handled in the `FigureSupplier` class, as per the checklist: 'You can pass random values to the constructor of a figure instead of generating them inside figure classes.'